linkerd/README.md

```diff
@@ -92,6 +92,10 @@ If you need to use a metric that is not provided by default, you can add an entr
 
 Simply follow the examples present in the [default configuration][4].
 
+**Note**: we append, in our code, linkerd metrics ending with '_total' to 'count' (ex. "linkerd.tcp.open_total" will be "linkerd.tcp.open.count")
```

Comment:
I don't think this section is necessary. However, I can submit a PR with the updated metric names, since this should have been added back when OpenMetricsV2 support was added for Linkerd. Once that PR is merged, it should be much more clear which metrics are being collected.

Reply:
https://github.com/DataDog/integrations-core/pull/12664

Reply:
Removed this part.
Thanks for the PR.